l10n-ms/src/main/java/com/example/l10nadmin/repository/L10nRepository.java

```diff
@@ -17,4 +20,11 @@
 
 	List<L10n> findByLocaleAndActiveIsTrue(String locale);
 
+	@Modifying
+	@Query(value = "update L10n set active=?5, locale=?2, lic=?3 ,value=?4,modifiedOn=?6 where id=?1", nativeQuery = true)
```

Comment:
😥
Why do this? Why not just `l10nRepository.save(entry)`?

Reply:
Let's keep the update SQL in order to be more performant and make the incoming DTO more tailored for the UI